### PR TITLE
Initial Confidential Ledger generated client

### PR DESCRIPTION
**Pull request by christothes**
Initial client and tests for Confidential Ledger

### CODE REVIEW

File: sdk/confidentialledger/Azure.Storage.ConfidentialLedger/api/Azure.Template.netstandard2.0.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+namespace Azure.Template
```
Comment (AlexGhiondea) [inline sdk/confidentialledger/Azure.Storage.ConfidentialLedger/api/Azure.Template.netstandard2.0.cs:1]: Is this correct?

Comment (christothes) [reply sdk/confidentialledger/Azure.Storage.ConfidentialLedger/api/Azure.Template.netstandard2.0.cs]: This was  just to see if you were paying attention :)

File: sdk/confidentialledger/Azure.Storage.ConfidentialLedger/README.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+portfolio, Azure Confidential Ledger runs in SGX enclaves. It is built on Microsoft Research's [Confidential Consortium Framework][ccf].
+
+  [Source code][client_src] | [Package (NuGet)][client_nuget_package] <!--| [API reference documentation][api_reference] | [Samples][samples] -->
+
+## Getting started
+
+This section should include everything a developer needs to do to install and create their first client connection *very quickly*.
+
+### Install the package
+Install the Azure Tables client library for .NET with [NuGet][table_client_nuget_package]:
```
Comment (pakrym) [inline sdk/confidentialledger/Azure.Storage.ConfidentialLedger/README.md:13]: nit: table_

File: sdk/confidentialledger/Azure.Storage.ConfidentialLedger/src/Azure.Storage.ConfidentialLedger.csproj [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This is a template project to demonstrate how to create a package that uses code generation as well as use for testing our build and release pipelines</Description>
+    <AssemblyTitle>Azure SDK Template</AssemblyTitle>
+    <Version>1.0.1-beta.1</Version>
+    <PackageTags>Azure Template</PackageTags>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <RootNamespace>Azure.Storage.ConfidentialLedger</RootNamespace>
```
Comment (pakrym) [inline sdk/confidentialledger/Azure.Storage.ConfidentialLedger/src/Azure.Storage.ConfidentialLedger.csproj:8]: rem

File: sdk/confidentialledger/Azure.Storage.ConfidentialLedger/src/Azure.Storage.ConfidentialLedger.csproj [hunk 4]
```diff
@@ -0,0 +1,39 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This is a template project to demonstrate how to create a package that uses code generation as well as use for testing our build and release pipelines</Description>
+    <AssemblyTitle>Azure SDK Template</AssemblyTitle>
+    <Version>1.0.1-beta.1</Version>
```
Comment (pakrym) [inline sdk/confidentialledger/Azure.Storage.ConfidentialLedger/src/Azure.Storage.ConfidentialLedger.csproj:5]:
```suggestion
    <Version>1.0.0-beta.1</Version>
```

File: sdk/confidentialledger/Azure.Storage.ConfidentialLedger/src/Azure.Storage.ConfidentialLedger.csproj [hunk 5]
```diff
@@ -0,0 +1,39 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This is a template project to demonstrate how to create a package that uses code generation as well as use for testing our build and release pipelines</Description>
+    <AssemblyTitle>Azure SDK Template</AssemblyTitle>
```
Comment (pakrym) [inline sdk/confidentialledger/Azure.Storage.ConfidentialLedger/src/Azure.Storage.ConfidentialLedger.csproj:4]: Fix title and description.

File: sdk/confidentialledger/Azure.Storage.ConfidentialLedger/src/autorest.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+title: ConfidentialLedger
+namespace: Azure.Storage.ConfidentialLedger 
+credential-types: TokenCredential
+credential-scopes: "https://confidential-ledger.azure.com/.default"
+low-level-client: true
+input-file:
+    -  https://github.com/Azure/azure-rest-api-specs/blob/0f79e4bb951c7dec6bfa03f856b39d03c4629101/specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/0.1-preview/common.json
+    -  https://github.com/Azure/azure-rest-api-specs/blob/0f79e4bb951c7dec6bfa03f856b39d03c4629101/specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/0.1-preview/confidentialledger.json
+    -  https://github.com/Azure/azure-rest-api-specs/blob/0f79e4bb951c7dec6bfa03f856b39d03c4629101/specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/0.1-preview/identityservice.json
+include-csproj: disable
```
Comment (pakrym) [inline sdk/confidentialledger/Azure.Storage.ConfidentialLedger/src/autorest.md:15]: shouldn't need this.

File: sdk/confidentialledger/Azure.Storage.ConfidentialLedger/src/properties/AssemblyInfo.cs [hunk 7]
```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+// Replace with test project/test project public key and uncomment to make internal members visible to
+// your test project. If not needed, this can be deleted.
+// [assembly: InternalsVisibleTo("Azure.Template.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100d15ddcb29688295338af4b7686603fe614abd555e09efba8fb88ee09e1f7b1ccaeed2e8f823fa9eef3fdd60217fc012ea67d2479751a0b8c087a4185541b851bd8b16f8d91b840e51b1cb0ba6fe647997e57429265e85ef62d565db50a69ae1647d54d7bd855e4db3d8a91510e5bcbd0edfbbecaa20a7bd9ae74593daa7b11b4")]
```
Comment (pakrym) [inline sdk/confidentialledger/Azure.Storage.ConfidentialLedger/src/properties/AssemblyInfo.cs:8]: remove?

**Review by pakrym — APPROVED** [review]

**Pull request merged**